src/styles/normalize.css

```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
Do not use `*` to reset something, just for box-sizing, count tags which u want reset

```
p,
h1,
... {
  m0
  p0
}
```